packages/aws-cdk-lib/aws-codebuild/lib/compute-type.ts

```diff
@@ -1,5 +1,7 @@
 /**
  * Build machine compute type.
+ *
+ * @see https://docs.aws.amazon.com/codebuild/latest/userguide/build-env-ref-compute-types.html#environment.types
```

Comment:
might be working combining this file, `environment-type.ts`, and `FleetComputeType` into an enums file?

Reply:
The current "convention" for this package is that most of the enums are part of their feature file (e.g. `ReportGroupType` in `report-group.ts`), with the exception of `ComputeType`, which has its own file. I assume this was because it was used across multiple "features", in this case both `Project` and `BuildImages`s, and now also `Fleet`. `EnvironmentType` is in a similar situation,  used both for the `BuildImage`s and `Fleet` classes.

If anything, the best thing to do would be to move both `ComputeType` and `EnvironmentType` in one of the feature files, probably `project.ts`, which contains `IBuildImage`?